packages/itwin/grouping-mapping-widget/src/widget/components/CalculatedPropertyActionWithVisuals.tsx

```diff
@@ -57,6 +60,19 @@ export const CalculatedPropertyActionWithVisuals = ({
   const [validator, showValidationMessage] = useValidator();
   const [resolvedHiliteIds, setResolvedHiliteIds] = useState<string[]>([]);
   const [colorProperty, setColorProperty] = useState<boolean>(false);
+  const origPropertyName = calculatedProperty?.propertyName ?? "";
+  const [showSaveValidationModal, setShowSaveValidationModal] = useState<boolean>(false);
+  const [invalidCustomCalcs, setInvalidCustomCalcs] = useState<invalids[]>([]);
+
+  async function checkOutliers(changedPropName: string): Promise<invalids[]> {
```

Comment:
Let's stay consistent and turn this into an arrow function.

Reply:
Added the arrow function to both of the "checkOutliers" functions